plasmapy/physics/transport.py

```diff
@@ -1,17 +1,78 @@
 """Functions to calculate transport coefficients."""
```

Comment:
"Large diffs are not displayed by default"

"But this is probably not a refactoring change, GitHub... oh."

Reply:
😕 confused... hwhat?

Reply:
Don't worry about it, just joking about the massive scope of this PR :smile: